README.md

```diff
@@ -42,7 +35,7 @@ Be sure to install [VS Code](https://code.visualstudio.com/Download) and create
 
 ![Browser your database data](media/browse-feature.png)
 
-### 4. Run queries
+### 3. Run queries
 
 1. Open the command palette with the keyboard shortcut `command` + `shift` + `p` if you are on a Mac (or `ctrl` + `shift` + `p` on a PC).
```

Comment:
```suggestion
1. Open the command palette with the keyboard shortcut `Cmd` + `Shift` + `P` if you are on a Mac (or `Ctrl` + `Shift` + `P` on a PC).
```

Reply:
@BrunoQuaresma I would also make this change to the command in item `#3`

Reply:
cc.: @n400

Reply:
It looked like the "Details" view in VS Code pulled its content from a different file... but I guess not... Now that I know that is not the case, I'll add another comment to this PR to put the link to the vsix installations instructions in the "Prerequisites" section.